laurat007/assignment2/AncestorsTest.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package assignment_2;
+
+import java.util.ArrayList;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import junit.framework.TestCase;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+/**
+ *
+ * @author Laura
+ */
+public class AncestorsTest extends TestCase{
+    
+    public AncestorsTest() {
+    }
+    
+    Ancestors.BinaryTreeNode<Integer> buildTree1() {
+        // build a binary tree
+        Ancestors.BinaryTreeNode<Integer> tree = new Ancestors.BinaryTreeNode<>(7);
+        tree.addLeft(3);
+        tree.addRight(4);
+        tree.addLeft(2);
+        tree.addBalanced(5, 0);
+        tree.addBalanced(8, 1);
+        tree.addBalanced(1, 1);
+        tree.addBalanced(6, 1);
+        return tree;
+    }
+    
+    @BeforeClass
+    public static void setUpClass() {
+    }
+    
+    @AfterClass
+    public static void tearDownClass() {
```

Comment:
I don't think you actually need those two @BeforeClass and @AfterClass methods at all :) They will be triggered and run but since you don't do anything in them you shouldn't even define them

Reply:
done.

Reply:
They were already there. It was not clear for me if they were supposed to be modified.